aucoalesce/coalesce.go

```diff
@@ -235,10 +245,23 @@ func normalizeCompound(msgs []*auparse.AuditMessage) (*Event, error) {
 			addSockaddrRecord(msg, event)
 		case auparse.AUDIT_EXECVE:
 			addExecveRecord(msg, event)
+		case auparse.AUDIT_AVC:
+			if keyCollision {
```

Comment:
What's the benefit of doing this conditionally vs always writing the data into avc / netfilter_cfg? Minimizing breaking changes when there are no duplicates?

Reply:
Really just reducing load on the index. If there are no collisions, we can get a normalisation. If there are collisions, we want to let the user make some sensible choice about how to deal with them and so provide the extra information.